src/uct/ib/ud/base/ud_ep.c

```diff
@@ -829,7 +884,15 @@ static void uct_ud_ep_do_pending_ctl(uct_ud_ep_t *ep, uct_ud_iface_t *iface)
 {
     uct_ud_send_skb_t *skb;
 
-    if (uct_ud_ep_ctl_op_check(ep, UCT_UD_EP_OP_CREQ)) {
+    if (ucs_unlikely(uct_ud_ep_ctl_op_check(ep, UCT_UD_EP_OP_FAIL))) {
```

Comment:
For the reason to have all pending control operations (managed by dumme element) handled in one place